erpnext/setup/doctype/naming_series/naming_series.py

```diff
@@ -25,14 +25,17 @@ def get_transactions(self, arg=None):
 		prefixes = ""
 		for d in doctypes:
 			options = ""
+			exist_flag = True
```

Comment:
better name for this is `found_all = True`

Still did not understand the use case?